spec/views/sign_up/emails/show.html.erb_spec.rb

```diff
@@ -23,4 +24,47 @@
 
     expect(rendered).to have_button(t('links.resend'))
   end
+
+  context 'when enable_load_testing_mode? is true and email address found' do
+    before do
+      allow(FeatureManagement).to receive(:enable_load_testing_mode?).and_return(true)
+      email_address = instance_double(EmailAddress, confirmation_token: 'some_token')
+      allow(EmailAddress).to receive(:find_with_email).with(email).and_return(email_address)
```

Comment:
don't stub the database model... create create a real row

```suggestion
      create(:email_ddress, confirmaiton_token: 'some_token')
```

Reply:
sure sir